app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.order(:name)
```

Comment:
Nice job implementing getting the ingredients sorted alphabetically but remember we will want to create a method in our `ingredients` model that handles the logic. Then we will just call that method here in our controller.